packages/rx_bloc_cli/mason_templates/bricks/feature_counter/__brick__/lib/l10n/network_lookup/counter_lookup.dart

```diff
@@ -21,8 +21,6 @@ class AppI18nCounterLookup extends I18nFeatureCounterLookup {
           bundledI18n = I18nFeatureCounter(I18nFeatureCounterLookup_bg());
           break;
         case 'en':
-          bundledI18n = I18nFeatureCounter(I18nFeatureCounterLookup_en());
```

Comment:
Please make sure such changes are addressed in separate PRs. From what I see this is not related to QR code integration, but feel free to correct me if I'm wrong

Reply:
These shouldn't necessarily be part of this PR, but since the same issue was pointed out by David for the QR implementation, and since the issue can be fixed quickly, I decided to make the change for every feature where the issue is present

Reply:
OK, in this case proceed with the applied changes. But keep in mind that according to the good practices the PR should contain changes related to the ticket only. Otherwise such can introduce unforeseen side effects